src/domains/nodo-common/03_cosmosdb_nodo_verifyko.tf

```diff
@@ -1,9 +1,16 @@
+resource "azurerm_resource_group" "nodo_verifyko_to_datastore_rg" {
+  name     = "${local.project}-verifyko-to-datastore-rg"
+  location = var.location
+
+  tags = var.tags
+}
+
 module "cosmosdb_account_nodo_verifyko" {
   source              = "git::https://github.com/pagopa/terraform-azurerm-v3.git//cosmosdb_account?ref=v6.7.0"
   domain              = var.domain
   name                = "${local.project}-verifyko-cosmos-account"
   location            = var.location
-  resource_group_name = azurerm_resource_group.db_rg.name
+  resource_group_name = azurerm_resource_group.nodo_verifyko_to_datastore_rg.name
```

Comment:
@aomegax check the verifyko_cosmos_nosql_db_params variable for prod env; the new account must be zone redundant in the primary region
also check the max throughput, is it necessary to have 20000 RU (that will be doubled in a multi-region configuration like this one)?

Reply:
note that adding zone redundancy forces re-creation of the resource